bundle/Command/DataAnonymizerCommand.php

```diff
@@ -157,11 +157,10 @@ protected function proceedWithAction(InputInterface $input, OutputInterface $out
         return false;
     }
 
-    protected function getDateFromPeriod()
+    protected function getDateFromPeriod(): \DateTimeImmutable
```

Comment:
It's better to have `use DateTimeImmutable` so it is simplified usage here in the code.